pyanaconda/ui/gui/spokes/datetime_spoke.py

```diff
@@ -311,7 +311,10 @@ def _initialize(self):
             log.warning("%s is not a valid timezone, falling back to default (%s)",
                         kickstart_timezone, DEFAULT_TZ)
             self._set_timezone(DEFAULT_TZ)
-            self._timezone_module.Timezone = DEFAULT_TZ
+            self._timezone_module.SetTimezoneWithPriority(
+                DEFAULT_TZ,
+                constants.TIMEZONE_PRIORITY_KICKSTART
```

Comment:
This should be handled in the module. I think you can actually remove this whole section with a kickstart timezone.

Reply:
I'm not really sure about that. This is partial kickstart support. The module does not verify that the timezone we're setting is valid. It just passes it through without validation. That's probably ok for all-automatic installs. However the GUI might need to have some sane value when it opens. I will make this a separate commit, test and see what it does...